src/components/MultiSelectGroup/MultiSelectGroup.tsx

```diff
@@ -0,0 +1,103 @@
+import { chakra, Stack } from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+
+import Fieldset from "../Fieldset/Fieldset";
+import { LayoutTypes } from "../../helpers/types";
+import MultiSelect, { MultiSelectWidths } from "../MultiSelect/MultiSelect";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
+
+export interface MultiSelectGroupProps {
+  children: React.ReactNode;
+  /** Additional className to use. */
+  className?: string;
+  id: string;
+  /** Optional prop to overwrite default behavior of the `MultiSelect` component, if nothing gets passed, it is set to `undefined`. */
+  isBlockElement?: boolean | undefined;
```

Comment:
Let's wait to hear what @bigfishdesign13 and @nypl-wluisi think about the `isBlockElement`. Doing this goes against the pattern set in the DS and it feels like a special case that can be updated to not confuse devs.

Reply:
@EdwinGuzman In `ButtonGroup`, `CheckboxGroup` and `RadioGroup` we include props that mirror some of the props on the child components.  Passing one of those props to the group component sets the value for the corresponding prop on the child components.  I felt that using that same pattern here for `isBlockElement` would make sense.  How do you see this implementation in `MultiSelectGroup` as going against the DS patterns?

Reply:
That pattern is fine. The issue is with declaring the boolean prop to be of type `boolean | undefined`.

Reply:
@isastettler @EdwinGuzman 
The logic for `isBlockElement` that was described on Friday is appropriate: the default behavior for column shoudl be `true` and the default behavior for row shoudl be `false`.  That said, I know there were some syntactical issues that required us to use `undefined` for values in some other components, but I am not sure if that is necessary here.

Reply:
@bigfishdesign13 I think we should get rid of the `isBlockElement` prop on this component and have that setting controlled by the prop value of `layout`. This will solve the problem Edwin is highlighting and also make the component easier to use from a dev perspective.

Reply:
Yes, I was thinking that as well.  Let’s go ahead and do that.  The `isBlockElement` value for the child `MultiSelect` components can be set based on the value of `layout`.